packages/atlas/src/views/notifications/Notifications.tsx

```diff
@@ -74,12 +39,13 @@ export const Notifications = () => {
           )}
         </Header>
         <div>
-          {NOTIFICATIONS.map((notification, idx) => (
+          {notifications.map((notification, idx) => (
             <StyledNotificationTile
               key={`notification-${notification.id}-${idx}`}
-              {...notification}
-              onSelect={() => toggleSelected(notification.id)}
+              notification={notification}
               selected={selectedNotifications.includes(notification.id)}
+              onSelect={() => toggleSelected(notification.id)}
+              onClick={() => markNotificationAsRead(notification.id)}
```

Comment:
I'm confused about this behavior and simultaneously couldn't find the right answer in the design. 
Currently, if I click NFT title, the tile notification is marked as read, but when I do this on the notification widget, nothing happens. 
Shouldn't we do the same on NFT widget?

Reply:
Yep, I missed that, fixed now